cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/LanguageFrontend.kt

```diff
@@ -89,6 +90,14 @@ abstract class LanguageFrontend(
         return init(this)
     }
 
+    /**
+     * Similar to [parse], this function returns a [TranslationResult], but rather than parsing
+     * source code, the function [init] is used to build nodes in the Node Fluent DSL.
+     */
+    fun buildTR(init: LanguageFrontend.() -> TranslationResult): TranslationResult {
```

Comment:
should we instead return a translation result in `build`?

Reply:
I think it would make sense to change `build`